OMCompiler/SimulationRuntime/fmi/export/openmodelica/fmu2_model_interface.h

```diff
@@ -97,6 +97,7 @@ typedef struct {
   fmi2Real* states_der;
   fmi2Real* event_indicators;
   fmi2Real* event_indicators_prev;
+  fmi2Real* input_real_derivative;
```

Comment:
Where do you allocate the array?

Reply:
this is fixed with this https://github.com/OpenModelica/OpenModelica/pull/8357/commits/48fa7bfcd9be9744b9fb69705a05827d5fd2b514